agenthub/codeact_agent/prompt.py

```diff
@@ -0,0 +1,222 @@
+from opendevin.sandbox.plugins import SWEAgentCommandsRequirement
+
+_SWEAGENT_BASH_DOCS = '\n'.join(
+    filter(
+        lambda x: not x.startswith('submit'),
+        SWEAgentCommandsRequirement.documentation.split('\n')
+    )
+)
+# _SWEAGENT_BASH_DOCS content below:
+"""
+open <path> [<line_number>] - opens the file at the given path in the editor. If line_number is provided, the window will be move to include that line
+goto <line_number> - moves the window to show <line_number>
+scroll_down - moves the window down {WINDOW} lines
+scroll_up - moves the window down {WINDOW} lines
+create <filename> - creates and opens a new file with the given name
+search_dir <search_term> [<dir>] - searches for search_term in all files in dir. If dir is not provided, searches in the current directory
+search_file <search_term> [<file>] - searches for search_term in file. If file is not provided, searches in the current open file
+find_file <file_name> [<dir>] - finds all files with the given name in dir. If dir is not provided, searches in the current directory
+edit <start_line>:<end_line>
+<replacement_text>
+end_of_edit - replaces lines <start_line> through <end_line> (inclusive) with the given text in the open file. The replacement text is terminated by a line with only end_of_edit on it. All of the <replacement text> will be entered, so make sure your indentation is formatted properly. Python files will be checked for syntax errors after the edit. If the system detects a syntax error, the edit will not be executed. Simply try to edit the file again, but make sure to read the error message and modify the edit command you issue accordingly. Issuing the same command a second time will just lead to the same error message again.
+"""
+
+_COMMAND_DOCS = (
+    '\nApart from the standard bash commands, you can also use the following special commands in bash:\n'
```

Comment:
> Apart from the standard bash commands

This seems to contradict the SYSTEM_MESSAGE below, where it is mentioned that bash commands shall be wrapped with `<execute_bash>` tags.

Reply:
modified the prompt for clarity